README.md

```diff
@@ -111,6 +111,13 @@ Path to a configuration file used during compilation, currently only used to
 specify tunables. If specified, it should usually use SBT settings to create
 the path. Defaults to `None` if not provided.
 
+If a file exists with the same name as the config option, but with an
+additional Daffodil version extension (e.g. `config.xml.daffodil390`), then
```

Comment:
Can this be flipped to config.daffodil390.xml ?
Otherwise IDEs will not provide XML assist for these files without painfully adding that daffodil390 files are XML files.

Reply:
Makes the logic a bit more complicated, but agreed that it's the right thing to do. PR updated to look for the files where the daffodil version is a secondary extension. And modified a test to ensure this works.